apps/web/e2e/example.spec.js

```diff
@@ -0,0 +1,5 @@
+const { test } = require("@playwright/test");
+
+test("basic test", async ({ page }) => {
+  await page.goto("https://playwright.dev/");
```

Comment:
Why are we navigating to `https://playwright.dev/`?

Reply:
@kilemensi This is a sample test, that can be changed.

Reply:
@kilemensi I did include this 👆🏽 in my initial test.

Reply:
Are you talking about the URL (`playwright.dev`) @nerdlet? If yes, then no, the URL should be `localhost:<port>` i.e. what you have now in this PR.

What am asking you to do is add what comes after that i.e.
```js
  const title = page.locator('.navbar__inner .navbar__title');
  await expect(title).toHaveText('Playwright');
```

Basically what are we _expected_ to find once we've navigated to that page? Going by current code in `main`, our check would be something like:
```js
  const title = page.locator('.styles.title');
  await expect(title).toHaveText('Web Welcome to Next.js');
```
Or something similar.

Reply:
@kilemensi Meant the port thing because if we do the above , we still need to specify the port. Let me add that then ping you